doc/api/dns.markdown

```diff
@@ -8,9 +8,9 @@ This module contains functions that belong to two different categories:
 
 1) Functions that use the underlying operating system facilities to perform
 name resolution, and that do not necessarily do any network communication.
-This category contains only one function: `dns.lookup`. __Developers looking
+This category contains only one function: `dns.lookup()`. __Developers looking
 to perform name resolution in the same way that other applications on the same
-operating system behave should use `dns.lookup`.__
+operating system behave should use [`dns.lookup()`](#dns_dns_lookup_hostname_options_callback).__
```

Comment:
When you are using the same thing again and again you can add the reference at the bottom.